internal/commands/run.go

```diff
@@ -83,19 +83,33 @@ func readYAMLConfig() (string, error) {
 	return string(yamlConfig), nil
 }
 
-func readStarlarkConfig(ctx context.Context, env map[string]string) (string, error) {
-	starlarkSource, err := ioutil.ReadFile(".cirrus.star")
+func readStarlarkConfig(ctx context.Context, path string, env map[string]string) (string, error) {
```

Comment:
Maybe rename it to `evaluateStarlarkConfig` or `readAndEvaluateStarlarkConfig` since it translating Starlark to YAML?

Reply:
Renamed in https://github.com/cirruslabs/cirrus-cli/pull/169/commits/fd2fe7e15488ce783b641f70fd660357b90f0893 to `EvaluateStarlarkConfig`.